socorro/signature/siglists_utils.py

```diff
@@ -27,25 +28,50 @@ class BadRegularExpressionLineError(Exception):
     """Raised when a file contains an invalid regular expression."""
 
 
-def build_filepath(source):
+class _PackageSource:
```

Comment:
I'm going to rename this to "INCLUDED" because that's what we're using in the comments and other places.